README.md

```diff
@@ -36,7 +36,9 @@ All contributors are required to abide by our [Code of Conduct](https://github.c
 ## All Contributors
 
 <!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
+
 [![All Contributors](https://img.shields.io/badge/all_contributors-129-orange.svg?style=flat-square)](#contributors-)
+
```

Comment:
You haven't run the `npm run contributors:generate`.

Also please remove the new lines (line 39 & 41).